src/app/components/buildings/goldmine/index.ts

```diff
@@ -0,0 +1 @@
+export * from './goldmine'
```

Comment:
We probably won't add new files in eg. house folder so there is no need to create it. Take a look:

Folder buildings/
Goldmine.ts
Wall.ts
House.ts
Index.ts

Index.ts (already done)
Export * from './goldmine.ts'
Etc...


The same for units

Reply:
We don't put there any e.g. css files?

Reply:
CSS files will be in content/css directory and we will load them at the very beginning